addons/mrp/models/mrp_workorder.py

```diff
@@ -283,15 +285,15 @@ def _assign_default_final_lot_id(self):
     def record_production(self):
         self.ensure_one()
         if self.qty_producing <= 0:
-            raise UserError(_('Please set the quantity you produced in the Current Qty field. It can not be 0!'))
+            raise UserError(_('Please set the quantity you are currently producing. It should be different from zero.'))
 
         if (self.production_id.product_id.tracking != 'none') and not self.final_lot_id:
             raise UserError(_('You should provide a lot/serial number for the final product'))
 
         # Update quantities done on each raw material line
```

Comment:
If the move is not tracked, directly update the consume quantity according to the produce quantity